Source/Game.cs

```diff
@@ -198,6 +202,14 @@ private void HandleError(Exception e)
 
 	public override void Update()
 	{
+		if (Input.Keyboard.Pressed(Keys.F5))
+		{
+			if (Scene is Startup || Scene is GameErrorMessage) return;
+
+			Log.Info($"--- User has initiated a{(Input.Keyboard.CtrlOrCommand ? " full" : String.Empty)} manual reload. ---");
+			ReloadAssets(Input.Keyboard.CtrlOrCommand); // F5 - Reload changed; Ctrl + F5 - Reload all
+		}
```

Comment:
We actually kind of already have a hotkey for reloading assets, where if you press Ctrl+R it will also call the ReloadAssets function. This actually comes from the base game.
It's a little bit different, because it doesn't let you filter to changed vs all like this, but we probably don't need both of these functions, since they're kind of doing the same thing.

Reply:
I'd be fine with replacing vanilla's ctrl+r with this implementation.